### ISSUE TITLE
Can't disconnect wallet and DEX using mob browser and app

### ISSUE THREAD

**Issue by Liubov-crypto** [aeternity/dex-ui#348]
Can't disconnect wallet and DEX:
https://user-images.githubusercontent.com/69896204/172427314-a82ba492-10e3-41f0-95df-93a44b505816.MP4

**Comment by Liubov-crypto** [comment]
I used for tests wallet: commit 8b85da9 / v.0.9.9 and browser: Safari ver. 15.5 

It seems that the connection with the exchange continues to work incorrectly. I logged out in one tab, which is hidden, but in the other, I continue to be connected to the exchange. 

I also had a test network selected in my wallet, but the exchange shows that I am using the main network:

![photo_2022-07-14 15 54 16](https://user-images.githubusercontent.com/69896204/178987099-0812d45b-0fb9-48fa-8e45-76700b1bb635.jpeg)

https://user-images.githubusercontent.com/69896204/178987104-e7a630c9-73b1-4ec1-98d9-92cffddc7f75.mp4
